app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
+      flash[:success] = t "sessions.login_success"
+      log_in user
+      redirect_to user_path id: user.id
+    else
+      flash[:danger] = t "sessions.login_failed"
```

Comment:
```suggestion
      flash.now[:danger] = t "sessions.login_failed"
```

Phân biệt flash và flash.now cho chị nghe nhé

Reply:
`flash[:danger]` sẽ tiếp tục với action tiếp theo và sẽ được sử dụng khi chuyển hướng đến một action khác thông qua phương thức `redirect_to`.
`flash.now[:danger]` sẽ được hiển thị trong view mà ta đang render thông qua method `render`.